tests/unit/disc/UnitTest_BlockedDOFManager.cpp

```diff
@@ -73,199 +73,201 @@ tests are a beginning, "work in progress."
       int numProcs = comm->getSize();
       int myRank = comm->getRank();
 
-      // panzer::pauseToAttach();
-
-      using Teuchos::RCP;
-      using Teuchos::rcp;
-      using Teuchos::rcp_dynamic_cast;
-
-      // Build an STK Discretization object that holds the test mesh "2D_Blk_Test.e"
-
-      // 2D, quad, 3 block mesh built in Cubit.
-      Teuchos::RCP<Teuchos::ParameterList> discParams = rcp(new Teuchos::ParameterList);
-      discParams->set<std::string>("Exodus Input File Name", "2D_Blk_Test.e");
-      discParams->set<std::string>("Method", "Exodus");
-      discParams->set<int>("Interleaved Ordering", 2);
-      discParams->set<bool>("Use Composite Tet 10", 0);
-      discParams->set<int>("Number Of Time Derivatives", 0);
-      discParams->set<bool>("Use Serial Mesh", 1);
-
-      // Need to test various meshes, with various element types and block structures.
+      if(numProcs==1) {
+         // panzer::pauseToAttach();
 
-      Teuchos::RCP<Albany::AbstractMeshStruct> meshStruct;
-      meshStruct = DiscretizationFactory::createMeshStruct(discParams, comm, 0);
-      auto ms = Teuchos::rcp_dynamic_cast<AbstractSTKMeshStruct>(meshStruct);
-      auto gms = Teuchos::rcp_dynamic_cast<GenericSTKMeshStruct>(meshStruct);
+         using Teuchos::RCP;
```

Comment:
That is not my code, this is the test which existed already before. I just added an if condition to prevent from testing it if run with MPI.

But I can clean a bit the lines if needed.